teams/hubturbo.md

```diff
@@ -30,7 +30,7 @@ permalink: /teams/hubturbo/
 
  - [@xinan](https://github.com/xinan)
    ([Liu Xinan](http://xinan.io))
-   - Technology Expertise:
+   - Technology Expertise: Performance Optimization
```

Comment:
General response time. So far it is pretty slow to refresh a repo, or open a new repo. I hope to optimise the speed for all these slow actions.